sensor/docs/tutorials/tag_based_sfm_calibrator.md

```diff
@@ -0,0 +1,360 @@
+
+Note: we assume the user knows how to use a camera and make no attempt at explaining camera fundamentals in this tutorial.
+
+Camera selection and usage recommendations:
+
+- It is recommended to use cameras with the highest (real) resolution possible and use image formats with as little compression as possible (DLSR cameras have options like normal, fine, ultra-fine, and so on).
+- It is recommended to use a wide lens to capture as much of the environment as possible. Otherwise, the camera can not capture a high number of tags in a single image. However, fish-eye lenses are not supported.
+- Use the highest F-number possible (smaller aperture). For example, we use at least f/14. This ensures that the tags will be in focus at the relevant distances.
+- Use a relatively high shutter speed. The specific number will depend on the user's ability, but while it is acceptable to have underexposed photos, it is not acceptable to have blurred ones (motion while exposing the sensor to light).
+- Regarding the ISO value, while in photography it is preferred to have low ISO values, in this problem setting (detecting tags), values over 1000 are not an issue (taking into consideration the fact that we require a small aperture and high shutter speeds, higher ISO values become necessary).
+- The use of flash is optional and its use is left up to the user
+- Even when using prime lenses, the focus ring modifies the camera intrinsics. For this reason, it is required to have the focus set to manual in both the camera lens (when applicable) and to set its value, it is recommended to either set it to infinity or first use auto-focus once and then fix the focus by setting it to manual.
+
+## Launching the tool
+
+In this tutorial, we will use the RDV of Tier IV (R&D Vehicle).
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `rdv`, and in `calibrator`, select `tag_based_sfm_base_lidar_calibrator`. Then, press `Continue`.
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient. However, for this tutorial, we will use the default values. After configuring the parameters, click `Launch`.
+
+The following UI should be displayed. When the `Calibrate` button becomes available, click it.
+If it does not become available, it means that either the required `tf` or services are not available. In this case, since the `tf` are published by the provided rosbags, run the first one for a few seconds and then pause it (`ros2 bag play scene0.db3 --clock -r 0.5`).
```

Comment:
Is the "first one" here indicate the rosbag?

Reply:
Thanks. Addressed in [233558e](https://github.com/tier4/CalibrationTools/pull/164/commits/233558e867b703832ccd0f553731195bef14bc3e)